graphiti_core/graph_queries.py

```
@@ -0,0 +1,131 @@
+            """CREATE FULLTEXT INDEX community_name IF NOT EXISTS 
+            FOR (n:Community) ON EACH [n.name, n.group_id]""",
+            """CREATE FULLTEXT INDEX edge_name_and_fact IF NOT EXISTS 
+            FOR ()-[e:RELATES_TO]-() ON EACH [e.name, e.fact, e.group_id]""",
+        ]
+    
+def get_nodes_query(name: str, query: str, db_type: str = "neo4j") -> str:
+    if db_type == "falkordb":
+        label = NEO4J_TO_FALKORDB_MAPPING[name]
+        return f"CALL db.idx.fulltext.queryNodes('{label}', {query})"
+    else:
+        return f'CALL db.index.fulltext.queryNodes("{name}", {query}, {{limit: $limit}})'
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)